### Задание 2.3.1. Оценка вероятности по неравенству Чебышева

**Условие:**

Пусть случайная величина $$ \eta_n $$ — это сумма очков, появившихся при $$ n $$ независимых подбрасываниях честной шестигранной игральной кости.

Оценить вероятность:

$$
P\left( \left| \frac{\eta_n}{n} - 3.5 \right| > \varepsilon \right)
$$

используя **неравенство Чебышёва**, при следующих значениях:

- $$ n = 10 $$
- $$ \varepsilon = 1 $$



#### Шаг 1. Теоретические значения

Для одной кости $$ X_i ,  i = 1, \dots, n $$

- Возможные значения: 1, 2, 3, 4, 5, 6
- Математическое ожидание:
  $$
  E[X_i] = \frac{1 + 2 + 3 + 4 + 5 + 6}{6} = 3.5
  $$

- Дисперсия:
  $$
  D[X_i] = E[X_i^2] - (E[X_i])^2 = \frac{1^2 + 2^2 + \dots + 6^2}{6} - 3.5^2 = \frac{91}{6} - 12.25 = \frac{35}{12}
  $$

Сумма $$ \eta_n = X_1 + \dots + X_n  $$ тогда:


-  $$ E[\eta_n] = n \cdot 3.5 $$
- $$ D[\eta_n] = n \cdot \frac{35}{12} $$

Выборочное среднее:
$$
\bar{X}_n = \frac{\eta_n}{n}
$$

Тогда дисперсия $$ \bar{X}_n : $$
$$
D\left[ \frac{\eta_n}{n} \right] = \frac{D[\eta_n]}{n^2} = \frac{n \cdot \frac{35}{12}}{n^2} = \frac{35}{12n}
$$


In [13]:
from sympy import Rational

# Дано:
n = 10
epsilon = 1

# Дисперсия одного броска
Dx = Rational(35, 12)

# Дисперсия среднего
D_mean = Dx / n

# Оценка по неравенству Чебышёва
P_upper = D_mean / epsilon**2
float(P_upper), round(P_upper.evalf(), 4)


(0.2916666666666667, 0.2917)

### Ответ:

Согласно неравенству Чебышева:

$$
P\left( \left| \frac{\eta_{10}}{10} - 3.5 \right| > 1 \right) \leq \frac{35}{120} \approx \boxed{0.2917}
$$

То есть, вероятность того, что среднее значение отклонится от математического ожидания больше чем на 1, **не превышает 29.17%**.


#### Задание 2.3.2. Закон больших чисел и независимые случайные величины

**Условие:**

Пусть дана последовательность независимых случайных величин $$ (\xi_k)_{k=1}^{\infty} $$ причём:

$$
P(\xi_k = \sqrt{k}) = P(\xi_k = -\sqrt{k}) = \frac{1}{2\sqrt{k}}, \quad
P(\xi_k = 0) = 1 - \frac{1}{\sqrt{k}}
$$

Нужно определить, **выполняется ли закон больших чисел** для этой последовательности $$ \xi_k $$



#### Закон больших чисел в форме Маркова

Если последовательность случайных величин ξₖ является независимой и удовлетворяет условиям:

- математическое ожидание E[ξₖ] существует (конечно),
- и выполнено:
  
  $$
  \sum_{k=1}^{\infty} \frac{\text{Var}[ξ_k]}{k^2} < \infty,
  $$

то выполняется **закон больших чисел** в форме Маркова для последовательности средних:

$$
\frac{1}{n} \sum_{k=1}^{n} ξ_k \to 0 \quad \text{(по вероятности)}
$$

---

### Шаг 1. Математическое ожидание и дисперсия

Каждая случайная величина ξₖ принимает три значения:

- ±√k с вероятностью 1 / (2√k),
- 0 с вероятностью 1 − (1 / √k).

---

#### Математическое ожидание:

$$
E[ξ_k] = \sqrt{k} \cdot \frac{1}{2\sqrt{k}} + (-\sqrt{k}) \cdot \frac{1}{2\sqrt{k}} + 0 \cdot \left(1 - \frac{1}{\sqrt{k}}\right) = 0
$$

---

#### Дисперсия:

$$
E[ξ_k^2] = (\sqrt{k})^2 \cdot \frac{1}{2\sqrt{k}} + (-\sqrt{k})^2 \cdot \frac{1}{2\sqrt{k}} = k \cdot \left( \frac{1}{\sqrt{k}} \right) = \sqrt{k}
$$

#### Значит:

$$
\text{Var}[ξ_k] = E[ξ_k^2] - (E[ξ_k])^2 = \sqrt{k}
$$


In [27]:
from sympy import symbols, Sum, sqrt, oo

k = symbols('k', integer=True, positive=True)
term = sqrt(k) / k**2  # Var[ξ_k] / k²
series = Sum(term, (k, 1, oo))
series.doit().evalf()


2.61237534868549

### Вывод:

Мы получили:

$$
\sum_{k=1}^\infty \frac{\text{Var}[\xi_k]}{k^2} = \sum_{k=1}^\infty \frac{\sqrt{k}}{k^2} = \sum_{k=1}^\infty \frac{1}{k^{3/2}}
$$

Это **сходящийся числовой ряд**, так как $$ p = \frac{3}{2} > 1 $$

---

### Следовательно:

По **закону больших чисел в форме Маркова**:
$$
\frac{1}{n} \sum_{k=1}^n \xi_k \xrightarrow{P} 0
$$

То есть, **закон больших чисел выполняется** для данной последовательности $$ \xi_k $$.
